project_billing_utils/invoice.py

```diff
@@ -44,29 +41,22 @@ def name_get(self, cr, uid, ids, context=None):
                 # We will
```

Comment:
you don't need  `read`. Something like the following should do the same (just check the bit with partner_id)

    for rec in self:
        rest.append((rec.id, ('%s - %s - %s' % (rec.number or '',
                                         rec.partner_id.name_get(), rec.name or ''))))